configurations/pipeline_configuration.py

```diff
@@ -0,0 +1,200 @@
+from core_data_modules.cleaners import somali
+from dateutil.parser import isoparse
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="RVI-ELECTIONS",
+    project_start_date=isoparse("2022-03-11T00:00:00+03:00"),
+    project_end_date=isoparse("2100-01-01T00:00:00+03:00"),
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/IMAQAL"
+    ),
+    uuid_table=UUIDTableClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+        table_name="avf-global-urn-to-participant-uuid",
+        uuid_prefix="avf-participant-uuid-"
+    ),
+    operations_dashboard=OperationsDashboardConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-dashboards-firebase-adminsdk-gvecb-ef772e79b6.json"
+    ),
+    rapid_pro_sources=[
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/imaqal-text-it-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                FlowResultConfiguration("RVI_elections_demog", "imaqal_pool_age", "age"),
+                FlowResultConfiguration("RVI_elections_demog", "imaqal_pool_district", "location"),
+                FlowResultConfiguration("RVI_elections_demog", "imaqal_pool_gender", "gender"),
+                FlowResultConfiguration("RVI_elections_demog", "imaqal_pool_recently_displaced", "recently_displaced"),
+
+                FlowResultConfiguration("RVI_elections_s01e01_activation", "rqa_rvi_elections_s01e01", "rvi_elections_s01e01"),
+            )
+        )
+    ],
+    coda_sync=CodaConfiguration(
+        coda=CodaClientConfiguration(credentials_file_url="gs://avf-credentials/coda-production.json"),
+        sync_config=CodaSyncConfiguration(
```

Comment:
If you're not ready to add configuration for the rqa yet, can you put a todo in instead?